### PR TITLE
Add quiz module

### PR DESCRIPTION
**Pull request by DEVTomatoCake**
Replaces #111 

(send help but no money pls)

This quiz module allows users to create playable quiz using `/quiz create` and `/quiz create-bool`, and additionally for trusted admins to create quiz which are playable using `/quiz play` privately to compete on a quiz leaderboard. Admins can create quiz, set emojis, the leaderboard channel, embed strings, how many quiz can be played per day and how a `/quiz play` quiz is selected.

### CODE REVIEW

File: locales/de.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
     "config": {
         "checking-config": "Konfiguration wird überprüft...",
         "done-with-checking": "Konfiguration erfolgreich überprüft.",
-        "creating-file": "Konfiguration %m/%f exestiert aktuell nicht, wird aber gleich erstellt...",
+        "creating-file": "Konfiguration %m/%f existiert aktuell nicht, wird aber gleich erstellt...",
```
Comment (SCDerox) [inline locales/de.json:63]: These changes won't be live to SCNX, please use weblate to change translations

File: locales/en.json [hunk 2]
```diff
@@ -910,5 +910,40 @@
...
         "previous-cards": "Previous cards: "
+    },
+    "quiz": {
+        "what-have-i-voted": "What have I voted?",
+        "vote": "Vote!",
+        "vote-this": "Select this option if you think it's correct.",
+        "voted-successfully": "Selected successfully.",
+        "not-voted-yet": "You have not selected an option yet, so I cant show you what you selected?",
+        "you-voted": "You've selected **%o** as correct answer.",
+        "change-opinion": "You can change your oponion at any time by selecting another option above the button you just clicked.",
```
Comment (SCDerox) [inline locales/en.json:921]:
```suggestion
        "change-opinion": "You can change your opinion at any time by selecting another option above the button you just clicked.",
```

**Review by SCDerox — CHANGES_REQUESTED** [review]
Thanks so much for your contribution.
Some small things - also another thing: Could you please add an option to disable live-previews in the configuration?

File: locales/de.json (hunk 1 above)
Comment (DEVTomatoCake) [reply locales/de.json:63]: I dont want them to be live, I just want them fixed locally xD

**Review by SCDerox — APPROVED** [review]
lgtm - thanks so much for your contribution :heart: 
Your changes should be available on SCNX shortly.

**Pull request merged**